README.md

```diff
@@ -96,3 +96,5 @@ SimpleCov.start
 ```
 
 You can see your test coverage when you run your tests. If you want this in a graphical form, uncomment the `HTMLFormatter` line and see what happens!
+
+Please see RPS_CF_Diagram.jpg (in RPS-CHALLENGE-APPRENTICESHIPS/docs) 
```

Comment:
Good diagram! Can see you have a clear understanding of how the request/response cycle works.